.github/workflows/bcny-firebase.yml

```diff
@@ -28,7 +29,6 @@ jobs:
         with:
           fetch-depth: 1
           path: ${{ github.workspace }}/SourceCache/firebase-cpp-sdk
-          ref: refs/heads/compnerd/swift
```

Comment:
One more question: Assuming that our local changes are all in the compnerd/swift branch, rather than main, wouldn't we still want to build compnerd/swift branch in workflow_dispatch case? Or do we build compnerd/swift branch by default? I'm not sure how that works

Reply:
CC @compnerd

Reply:
From [these docs](https://github.com/actions/checkout), if we remove `ref` then falls back to using the reference git sha (the PR branch HEAD on `pull_request`) or the default branch `compnerd/swift` on `push` or `workflow_dispatch`:

```
# The branch, tag or SHA to checkout. When checking out the repository that
# triggered a workflow, this defaults to the reference or SHA for that event.
# Otherwise, uses the default branch.
```

Reply:
Thanks for looking that up!